ansible/roles/vm_set/files/mux_simulator.py

```diff
@@ -268,18 +269,29 @@ def set_active_side(vm_set, port_index, new_active_side):
     Args:
         vm_set (string): The vm_set of test setup.
         port_index (int or string): Index of the port.
-        new_active_side (string): Either "tor_a" or "tor_b".
+        new_active_side (string): One of: 'tor_a', 'tor_b', 'toggle', 'random'. If new_active_side is 'toggle',
+            always toggled the active side. If new_active_side is 'random', randomly choose new side from 'tor_a'
+            and 'tor_b'.
```

Comment:
What's the purpose for toggling to a random port?

Reply:
I believe that in production, not all mux cables have the same active side. This is to simulate the possible state in production.